x-team/xp-ui
# Create Tagslist Component to be used on JobCards and JobDetailsPage

Fixes https://x-team-internal.atlassian.net/browse/XP-3366

## Description

Creates TagsList component, which is being implement into JobCard and JobDetails Page, replacing code that was rendering the tags in the latter.

## Checklist


## Related PRs

branch | PR
------ | ------

xp_registration_branch_name | [XP Registration](https://github.com/x-team/xp-registration/pull/X)

## Steps to Test or Reproduce

Check My Applications page once this PR and the PR open on xp-registration are merged

## Impacted Areas in Application

My Applications Page, Job Details page

## Screenshots

![Screen Shot 2019-11-14 at 10 34 07 AM](https://user-images.githubusercontent.com/611845/68889416-4143bf80-06f3-11ea-8b93-97bece814c26.png)
